.github/wiki/Deliverable-Configurations.md

```diff
@@ -0,0 +1,15 @@
+## Deliverable Configurations
+
+The file [DeliverableConstants.rb](https://github.com/nimblehq/ios-templates/blob/develop/fastlane/Constants/DeliverableConstants.rb) contains the settings for build delivery, including:
+- Firebase
+- App Store
+
+## Use the template
+
+1. Running the `make.sh` script will ask if the developer wants to configure the `DeliverableConstants` file.
+2. When confirming with the prompt, the template will launch Xcode to modify the `DeliverableConstants` file.
+
+## Configure later
```

Comment:
Please add a new line after this section

Reply:
Fix in https://github.com/nimblehq/ios-templates/pull/320/commits/9802c59357c1203fdcc9b21e5af0f57f0109237a